### PR TITLE
Enable all log domains and map it with existing domains

### PR DESCRIPTION
**Pull request by jayahariv**
SQL = Database;
Changes = Replicator;
BLIPMessages = Network;
TLS = Network
Zip=Network

### CODE REVIEW

File: Objective-C/CBLLog.mm [hunk 1]
```diff
@@ -166,6 +169,11 @@ - (instancetype) initWithDefault {
         kCBL_LogDomainListener  = setNamedLogDomainLevel("Listener", kC4LogDebug);
         setNamedLogDomainLevel("BLIP", kC4LogDebug);
         setNamedLogDomainLevel("SyncBusy", kC4LogDebug);
+        setNamedLogDomainLevel("TLS", kC4LogDebug);
+        setNamedLogDomainLevel("Changes", kC4LogDebug);
+        setNamedLogDomainLevel("SQL", kC4LogDebug);
+        setNamedLogDomainLevel("Zip", kC4LogDebug);
```
Comment (jayahariv) [inline Objective-C/CBLLog.mm:175]: Zip and TLS > enabled for default Database domains?

**Comment by pasin** [comment]
TLS is NETWORK.

File: Objective-C/CBLLog.mm [hunk 2]
```diff
@@ -81,11 +81,15 @@ static C4LogDomain setNamedLogDomainLevel(const char *domainName, C4LogLevel lev
...
         domainDictionary = @{ @"DB": @(kCBLLogDomainDatabase),
+                              @"SQL": @(kCBLLogDomainDatabase),
                               @"Query": @(kCBLLogDomainQuery),
                               @"Sync": @(kCBLLogDomainReplicator),
                               @"SyncBusy": @(kCBLLogDomainReplicator),
+                              @"Changes": @(kCBLLogDomainReplicator),
                               @"BLIP": @(kCBLLogDomainNetwork),
                               @"WS": @(kCBLLogDomainNetwork),
+                              @"BLIPMessages": @(kCBLLogDomainNetwork),
+                              @"Zip": @(kCBLLogDomainNetwork),
```
Comment (jayahariv) [inline Objective-C/CBLLog.mm:92]:
```suggestion
                              @"Zip": @(kCBLLogDomainNetwork),
                              @"TLS": @(kCBLLogDomainNetwork),
```

File: Objective-C/CBLLog.mm [hunk 3]
```diff
@@ -81,11 +81,16 @@ static C4LogDomain setNamedLogDomainLevel(const char *domainName, C4LogLevel lev
...
```
Comment (pasin) [inline Objective-C/CBLLog.mm:92]: MISSING TLS which should be in Network domain.

**Review by pasin — CHANGES_REQUESTED** [review]

**Comment by pasin** [comment]
Once this is merged, please share the change to the other platform.

File: Objective-C/CBLLog.mm [hunk 4]
```diff
@@ -81,11 +81,16 @@ static C4LogDomain setNamedLogDomainLevel(const char *domainName, C4LogLevel lev
...
+                              @"TLS": @(kCBLLogDomainNetwork),
```
Comment (jayahariv) [inline Objective-C/CBLLog.mm:93]: ^^^

**Review by pasin — APPROVED** [review]

**Pull request merged**